Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -690,6 +690,24 @@ void VertexShaderManager::ResetView()
 	bProjectionChanged = true;
 }
 
+void VertexShaderManager::TransformToClipSpace(const float* data, float *out)
+{
+	const float *world_matrix = (const float *)xfmem.posMatrices + g_main_cp_state.matrix_index_a.PosNormalMtxIdx * 4;
```

Comment:
It's safe to assume here that we are using the  global PosNormalMtxIdx and not a per vertex mtx idx?